specs/001-bible-transcription-site/contracts/search-api.md

```
@@ -0,0 +1,109 @@
+  request_id: string;
+}
+```
+
+## Contract Tests
+
+### Test Cases
+1. **Valid text search**
+   - Query: "love"
+   - Expected: 200 response with relevant verses
+
+2. **Valid reference search**
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)